com/AlterETM.py

```diff
@@ -103,26 +103,16 @@ def insert_modify_param(parser, cnn, stnlist, args):
 
     # create a bunch object to save all the params that will enter the database
     tpar = Bunch()
-    tpar.NetworkCode = None
-    tpar.StationCode = None
-    tpar.soln        = None
-    tpar.object      = None
-    tpar.terms       = None
-    tpar.frequencies = None
-    tpar.jump_type   = None
-    tpar.relaxation  = None
-    tpar.Year        = None
-    tpar.DOY         = None
-    tpar.action      = None
+    tpar.NetworkCode = tpar.StationCode = tpar.soln = tpar.object = tpar.terms = tpar.frequencies = None
```

Comment:
this is still a bit too long for pep8-- can you change this and the next line to:
```python
tpar.NetworkCode, tpar.StationCode, tpar.soln, tpar.object = None
tpar.terms, tpar.frequencies, tpar.jump_type, tpar.relaxation = None
tpar.Year, tpar.DOY, tpar.action = None
```

Reply:
a, b = None is not working
```
>>> a, b = None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: cannot unpack non-iterable NoneType object
>>> a, b = None, None
>>> a = b = None

```